webapp/src/client.ts

```diff
@@ -484,7 +490,8 @@ export default class CallsClient extends EventEmitter {
         this.closed = true;
         if (this.peer) {
             this.getStats().then((stats) => {
-                sessionStorage.setItem('calls_client_stats', JSON.stringify(stats));
+                const storage = window.desktop ? localStorage : sessionStorage;
+                storage.setItem(STORAGE_CALLS_CLIENT_STATS_KEY, JSON.stringify(stats));
```

Comment:
I didn't want to overcomplicate it by adding listeners, so the stats are saved when the client disconnects (user leaves call) and can be retrieved after that. 

This slightly differs from Web browser behavior, where stats can be fetched at any time during or after a call, but the simple fix is enough to cover the customer's use case (troubleshooting).